docs/docs/connectors/slack.mdx

```diff
@@ -12,7 +12,7 @@ Once you have them you can add these to your `credentials.yml`.
 
 **How to get the Slack credentials:** You need to set up a Slack app.
 
-1. To create the app go to [https://api.slack.com/apps](https://api.slack.com/apps) and click
+1. To create the app go to [Your Apps](https://api.slack.com/apps) and click
    on **Create New App**.
```

Comment:
I think we said we were going to mark up App functions with `<mark>` tags, like this `==Create New App==`

Reply:
Noted, I'm going to come back to this though